Common subdirectories: mercury-tests-10.04.orig/hard_coded/CVS and mercury-tests-10.04/hard_coded/CVS
Common subdirectories: mercury-tests-10.04.orig/hard_coded/exceptions and mercury-tests-10.04/hard_coded/exceptions
Common subdirectories: mercury-tests-10.04.orig/hard_coded/purity and mercury-tests-10.04/hard_coded/purity
Common subdirectories: mercury-tests-10.04.orig/hard_coded/sub-modules and mercury-tests-10.04/hard_coded/sub-modules
Common subdirectories: mercury-tests-10.04.orig/hard_coded/typeclasses and mercury-tests-10.04/hard_coded/typeclasses
diff -u mercury-tests-10.04.orig/hard_coded/write_reg1.exp3 mercury-tests-10.04/hard_coded/write_reg1.exp3
--- mercury-tests-10.04.orig/hard_coded/write_reg1.exp3	2007-02-22 18:35:31.000000000 +1300
+++ mercury-tests-10.04/hard_coded/write_reg1.exp3	2010-07-24 11:57:06.000000000 +1200
@@ -23,8 +23,8 @@
 'a'
 '&'
 3.14159
-1.12832498300000e-21
-2.23954899000000e+23
+1.128324983e-21
+2.23954899e+23
 -65
 4
 univ_cons(["hi! I\'m a univ!"])
diff -u mercury-tests-10.04.orig/hard_coded/write_xml.exp2 mercury-tests-10.04/hard_coded/write_xml.exp2
--- mercury-tests-10.04.orig/hard_coded/write_xml.exp2	2007-02-22 18:35:31.000000000 +1300
+++ mercury-tests-10.04/hard_coded/write_xml.exp2	2010-07-24 11:56:36.000000000 +1200
@@ -25,7 +25,7 @@
 			<Int type="int">123456</Int>
 			<Tuple--2--Tag_-123character-44-32float-125 functor="{}" type="{character, float}" arity="2">
 				<Char type="character">a</Char>
-				<Float type="float">1.23553225220000e-97</Float>
+				<Float type="float">1.2355322522e-97</Float>
 			</Tuple--2--Tag_-123character-44-32float-125>
 		</Tuple--3--Tag_-123string-44-32int-44-32-123character-44-32float-125-125>
 	</a_tuple--1--write_xml-46mytype>
@@ -404,7 +404,7 @@
 						<Int type="int">123456</Int>
 						<Tuple functor="{}" type="{character, float}" arity="2">
 							<Char type="character">a</Char>
-							<Float type="float">1.23553225220000e-97</Float>
+							<Float type="float">1.2355322522e-97</Float>
 						</Tuple>
 					</Tuple>
 				</a_tuple>
